### ISSUE TITLE
[Request] Bring back the drift speed boost

### ISSUE THREAD

**Issue by Neuron114** [PavelZinchenko/event-horizon-frontier-builds#223]
I notice that the mechanic where you gain extra speed when your ship is drifting is gone, so I personally I wish for it for that mechanic to be back because it's a cool mechanic to have especially when you need to avoid those drones with a very high speed 

but I'll let people vote for this.

**Comment by Inedilarion** [comment]
I don't think this bug has gone away. It was caused by the presence of enormous acceleration at a limited speed, and allowed one to overcome this speed limit (20). And now the limit has become much larger, and it has become more difficult to gain such a huge acceleration relative to it, so it has become almost impossible to use the bug. Well, the difficulty of carrying out a perfect drift has increased significantly due to the increase in the speed of the ship and the sharpness of its turn

**Comment by PavelZinchenko** [comment]
It wasn't a bug, it was initially designed to aid in exploration due to the long distances between locations. It allowed for gradual acceleration beyond the ship's speed. But now, with the increased speed limit, I don't think it's still necessary.
